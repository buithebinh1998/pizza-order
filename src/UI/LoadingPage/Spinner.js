import React from "react";
import "./LoadingPage.css";

const Spinner = () => {
  return (
    <div className="loading-page">
      <div className="loader" style={{ borderTopColor: "purple" }}></div>
    </div>
  );
};

export default Spinner;
