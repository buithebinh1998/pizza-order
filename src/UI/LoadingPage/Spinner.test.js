import { testSnapshots } from "../../utils/test.util";

import Spinner from "./Spinner";

describe("<Spinner / >", () => {
  testSnapshots(Spinner, [
    {
      props: {},
      description: "default render"
    }
  ]);
});
