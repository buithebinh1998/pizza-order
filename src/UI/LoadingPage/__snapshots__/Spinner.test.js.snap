// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner / > snapshots default render 1`] = `
<div
  className="loading-page"
>
  <div
    className="loader"
    style={
      Object {
        "borderTopColor": "purple",
      }
    }
  />
</div>
`;
